.github/workflows/build.yml

```diff
@@ -72,3 +74,30 @@ jobs:
       run: |
         docker push ${{env.DOCKER_REPO}}:${{env.BRANCH_TAG}}
         docker push ${{env.DOCKER_REPO}}:${{env.SHA_TAG}}
+
+    - name: Pin Terraform version
+      uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 0.12.29
+
+    - name: Install Terraform CloudFoundry Provider
+      run: |
+          mkdir -p $HOME/.terraform.d/plugins/linux_amd64 
+          cd $HOME/.terraform.d/plugins/linux_amd64
+          wget -q -O zipfile https://github.com/cloudfoundry-community/terraform-provider-cloudfoundry/releases/download/v0.12.4/terraform-provider-cloudfoundry_0.12.4_linux_amd64.zip && unzip zipfile && rm zipfile
+          chmod +x ${{ env.CF_PROVIDER }}
+          ls -lhat
+      env:
+        CF_PROVIDER: $HOME/.terraform.d/plugins/linux_amd64/terraform-provider-cloudfoundry_v0.12.4
+
+    - name: Terraform init
+      run: |
+          echo ${{ env.PAAS_TF_DIR }}
+          cd ${{ env.PAAS_TF_DIR }}
+          terraform init -backend-config=workspace-variables/qa_backend.tfvars
+    - name: Terraform apply
+      run: |
+          echo ${{ env.PAAS_TF_DIR }}
```

Comment:
Do we need this? The nice thing about GitHub Actions is that by default it shows us the commands being run in the runner UI.

Reply:
done